src/common/resources/voucher.js

```diff
@@ -27,8 +28,18 @@ export const VOUCHER_NUMBER_START = {
   ...BASE_RESOURCE,
   path: VOUCHER_NUMBER_START_API,
   POST: {
-    path: (queryParams, pathComponents, resourceData) => {
-      const sequenceNumber = get(resourceData, ['voucher_number', 'records', 0, 'sequenceNumber']);
+
+    path: (queryParams, pathComponents, resourceData, categories, stripesParams) => {
+      const { stripes } = stripesParams;
+
+      const form = getFormValues('configForm')(stripes.store.getState()) || {};
```

Comment:
this looks weird. Please explain the purpose of this code

Reply:
This need for get current value for reset voucher start number.

Reply:
It looks overcomplicated and not obvious, IMHO. To me usage of local resource looks more easy to understand. https://github.com/folio-org/stripes-connect/blob/master/doc/api.md#local-resources
You just use local resource in path as variable and set that local resource on button Reset, as well as call post on required resource.